src/app/connections/list-toolbar/list-toolbar.component.html

```diff
@@ -45,7 +45,7 @@
           </button>
       </div>
       <div class="toolbar-pf-action-right">
-        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['create']">Create
+        <button *ngIf="showCreate" type="button" class="btn btn-primary" [routerLink]="['/connections/create']">Create
```

Comment:
Yeah that doesn't work but I didn't want to be rude and remove it lol. I'll update that now. Grabbing a drink and then pushing up the changes for the Choose Connections and Configure Actions pages (search bar) and styling. Don't merge yet, please. Almost done!